src/main/java/mate/jdbc/model/Driver.java

```diff
@@ -0,0 +1,55 @@
+package mate.jdbc.model;
+
+public class Driver {
+    private Long id;
+    private String name;
+    private String licenseNumber;
+
+    public Driver() {
+
+    }
+
+    public Driver(Long id, String name, String licenseNumber) {
+        this.id = id;
+        this.name = name;
+        this.licenseNumber = licenseNumber;
+    }
+
+    public Long getId() {
+        return id;
+    }
+
+    public void setId(Long id) {
+        this.id = id;
+    }
+
+    public String getName() {
+        return name;
+    }
+
+    public void setName(String name) {
+        this.name = name;
+    }
+
+    public String getLicenseNumber() {
+        return licenseNumber;
+    }
+
+    public void setLicenseNumber(String licenseNumber) {
+        this.licenseNumber = licenseNumber;
+    }
+
+    @Override
+    public String toString() {
+        return "Driver{"
+                + "id="
+                + id
+                + ", name='"
+                + name
+                + '\''
+                + ", licenseNumber='"
+                + licenseNumber
+                + '\''
+                + '}';
```

Comment:
why so many new lines?
```suggestion
        return "Driver{"
                + "id=" + id
                + ", name='" + name  + '\''
                + ", licenseNumber='" + licenseNumber + '\''
                + '}';
```